assets/sass/local.sass

```diff
@@ -387,19 +387,15 @@ dt.summary-list__key__wider
 .width-half
   width: 50% !important
 
-.added-involved-staff
-  th
-    border-bottom: none
-  td
-    border-bottom: none
-
-.email
+.found-staff-detail
   padding-top: 8px
+  padding-bottom: 17px !important
```

Comment:
As we're not following govuk standard designs this is required.
We were using standard spacing but items slowly get out of line when large numbers of results are returned